include/zephyr/devicetree.h

```diff
@@ -2092,6 +2092,20 @@
  */
 #define DT_FOREACH_STATUS_OKAY_NODE(fn) DT_FOREACH_OKAY_HELPER(fn)
 
+/**
+ * @brief Invokes "fn" with arguments for every status "okay" node in the tree.
+ *
+ * The macro "fn" can take extra parameters except node identifier.
+ * The macro is expanded once for each node in the tree with status
+ * "okay" (as usual, a missing status property is treated as status
+ * "okay"). The order that nodes are visited in is not
+ * specified.
+ *
+ * @param fn macro to invoke
+ * @param ... variable number of arguments to pass to fn
+ */
+#define DT_FOREACH_STATUS_OKAY_VARGS_NODE(fn, ...) DT_FOREACH_OKAY_VARGS_HELPER(fn, __VA_ARGS__)
```

Comment:
1. Rename this to DT_FOREACH_STATUS_OKAY_NODE_VARGS. It is the "VARGS" version of DT_FOREACH_STATUS_OKAY_NODE. It is not the "NODE" version of DT_FOREACH_STATUS_OKAY_VARGS.
2. Add test coverage in tests/lib/devicetree/api/src/main.c

Reply:
okay, will update it.